presto-product-tests/bin/locations.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Since this is only sourced and not executed directly, it doesn't need a shebang line.